pkg/postgres/pgutils/utils.go

```diff
@@ -62,6 +63,18 @@ func NilOrTime(t *types.Timestamp) *time.Time {
 	return &ts
 }
 
+// NilOrUUID allows for a proto timestamp to be stored a timestamp type in Postgres
```

Comment:
I'm not sure if `nil` is good to have in our system. How about 0 uuid?

Reply:
@janisz I went with nil because I wanted a postgres error if we tried to stick an empty UUID in there.  If we go with a 0 uuid, then we have to add logic to ensure we aren't using a 0 uuid in a required field.  We also have the case of nullable columns where we may in fact want a null.  So allowing for a null to go into those columns seemed simpler.  That was my thought process.  I'll gladly discuss further if you think that is necessary.